app/models/labware_creators.rb

```diff
@@ -37,31 +37,38 @@ def custom_form?
       false
     end
 
-    # limber_plate_children (Plate -> Plate) (plate_creation#create)
-    # limber_plate_tubes (Plate -> Tube) (tube_creation#create)
-    # limber_plate_tube_racks (Plate -> TubeRack) (tube_rack_creation#create)
-    # limber_tube_children (Tube -> Plate) (nothing - want to be plate_creation#create)
-    # limber_tube_tubes (Tube -> Tube) (tube_creation#create)
-    # limber_tube_tube_racks (Tube -> TubeRack) (tube_rack_creation#create)
+    # plate_children (Plate -> Plate) (plate_creation#create)
+    # plate_tubes (Plate -> Tube) (tube_creation#create)
+    # plate_tube_racks (Plate -> TubeRack) (tube_rack_creation#create)
+    # tube_children (Tube -> Plate) (nothing - want to be plate_creation#create)
+    # tube_tubes (Tube -> Tube) (tube_creation#create)
+    # tube_tube_racks (Tube -> TubeRack) (tube_rack_creation#create)
```

Comment:
Looks like the above list is out of date? 
Shouldn't it be:
`plate_child_plate_creation`
etc.?

Could also do with a header comment above this list explaining what the list is.